editions/tw5.com/tiddlers/concepts/DefaultTiddlers.tid

```diff
@@ -1,10 +1,15 @@
 created: 20180306113011255
-modified: 20180306161225140
+modified: 20241104150049419
 tags: Concepts
 title: DefaultTiddlers
 type: text/vnd.tiddlywiki
 
-"Default tiddlers" refers to the list of tiddlers that is opened at startup in the browser if the address bar does not include a [[permalink|PermaLinks]]).
+"Default tiddlers" refers to the list of tiddlers that are opened at startup in the browser if the address bar does not include a [[permalink|PermaLinks]].
+
+There are two ways default tiddlers can be defined:
+
+* A [[title-list|Title List]] eg: `TiddlerTitles` and `[[Titles with spaces]]`
+* Filter expressions, using [[filter operators|Filter Operators]] eg: `[tag[HelloThere]]`
 
 At startup, the text of the system tiddler [[$:/DefaultTiddlers]] is retrieved and evaluated as a [[filter expression|Filter Expression]]. The resulting list of titles is then inserted into the [[story river|Story River]].
```

Comment:
I feel that we already explain here that a filter expression can be used to specify the default tiddlers. Perhaps we should consider rewriting this paragraph to further emphasize that, rather than add the paragraph above?

Reply:
>Perhaps we should consider rewriting this paragraph to further emphasize that, rather than add the paragraph above?

The paragraph has been there for years and extracting the information that DefaultTiddlers can handle filter expressions is not obvious and hard to find, as can be seen at the [discussion at Talk](https://talk.tiddlywiki.org/t/how-to-open-the-newly-edited-1-3-tiddler-in-the-story-river-every-time-i-open-the-wiki/11096/4)

A bullet-list attracts attention to the points that are important. I did not want to remove the paragraph, because I think its OK to explain functions in different ways for different users.